features/step_definitions/machine_misc.rb

```diff
@@ -26,4 +26,14 @@
      logger.warn "We will skip this scenario"
      skip_this_scenario
   end
-end 
+end
+
+Given(/^the last provisioned machine is worker$/) do
+  machines = BushSlicer::MachineMachineOpenshiftIo.list(user: admin, project: project("openshift-machine-api"))
+  cache_resources *machines
+  unless (machines.max_by(&:created_at).name.include? 'worker')
+     logger.warn "When no worker machine is present alerting ex-OCP37264 is not valid to test"
```

Comment:
May need to update this log, so other cases also can be used

Reply:
modified to make it look better , just wanted to have bit specific so that later we can add more cases as and when fail due to this requirement.